src/components/QueryHeader.tsx

```diff
@@ -157,6 +162,17 @@ export function QueryHeader({
           options={QUERY_FORMAT_OPTIONS}
         />
 
+        {editorMode === EditorMode.Code && (
+          <InlineSwitch
```

Comment:
more of a nit, but this setting looks a bit different / bigger compared to the other ones in the query header:


<img width="699" height="68" alt="Screenshot 2025-09-29 at 11 22 47 AM" src="https://github.com/user-attachments/assets/e2e240ba-2e19-4552-b56a-bcc525f6378c" />

Reply:
Good catch. I made some changes related to that.

<img width="702" height="244" alt="Screenshot 2025-09-30 at 11 38 34" src="https://github.com/user-attachments/assets/cbc59ea4-2b33-4457-b507-fcb1ba0d579e" />